code/bio_alias.h

```diff
@@ -70,6 +70,18 @@
 #define BIO_DISCARD_FLAG        REQ_OP_DISCARD
 #define BIO_IS_WRITE(bio)       op_is_write(bio_op(bio))
 
+#elif LINUX_VERSION_CODE >= KERNEL_VERSION(5, 5, 0) \
+  && LINUX_VERSION_CODE < KERNEL_VERSION(5, 5, 3)
+
+#define BI_RW                   bi_opf
+#define BI_DISK                 bi_disk
+#define BI_SIZE                 bi_iter.bi_size
+#define BI_SECTOR               bi_iter.bi_sector
+#define BIO_ENDIO(bio, err)     bio_endio(bio)
+#define BIO_IO_ERR(bio, err)    bio_io_error(bio)
+#define BIO_DISCARD_FLAG        REQ_OP_DISCARD
+#define BIO_IS_WRITE(bio)       op_is_write(bio_op(bio))
```

Comment:
Could you make this an extra clause on the 4.14-4.16 kernels above? All the fields appear to be the same so there doesn't seem to be a reason to duplicate them. You can use `||` in the `elif` for multiple disjoint kernel ranges like [here](https://github.com/utsaslab/crashmonkey/blob/master/code/bio_alias.h#L24).

Reply:
Yup! My bad didn't notice that